src/Enterspeed.Source.UmbracoCms/App_Plugins/Enterspeed.Dashboard/Dashboard.Sub.Views/failedjobs.view.html

```diff
@@ -1,49 +1,88 @@
+                    <div class="dashboard-list-item-property" style="width:20%">{{job.createdAt | date:'yyyy-MM-dd
+                        HH:mm:ss'}}
+                    </div>
+                    <div class="dashboard-list-item-property" style="width:20%">{{job.updatedAt | date:'yyyy-MM-dd
+                        HH:mm:ss'}}
+                    </div>
+                </div>
+                <div class="dashboard-list-item">
+                    <div class="dashboard-list-item-exception" style="width: 100%"
+                         ng-show="$index === vm.activeException">
+                        <h4>Exception</h4>
+                        {{job.exception === undefined || job.exception === null ? 'No exception' : job.exception}}
+                    </div>
+                </div>
+            </li>
+        </ul>
 
-		<div ng-if="vm.pagination.totalPages > 1 && !vm.dashboard.loading" class="flex justify-center ">
-			<umb-pagination page-number="vm.pagination.pageNumber"
-							total-pages="vm.pagination.totalPages"
-							on-next="vm.nextPage"
-							on-prev="vm.prevPage"
-							on-go-to-page="vm.goToPage">
-			</umb-pagination>
-		</div>
-	</div>
+        <div ng-if="vm.pagination.totalPages > 1 && !vm.dashboard.loading" class="flex justify-center ">
+            <umb-pagination page-number="vm.pagination.pageNumber"
+                            total-pages="vm.pagination.totalPages"
+                            on-next="vm.nextPage"
+                            on-prev="vm.prevPage"
+                            on-go-to-page="vm.goToPage">
+            </umb-pagination>
+        </div>
+        <div>
+            <div class="seed-dashboard-text block-form">
+                <br>
+                <h4>Delete failed jobs</h4>
+                <br>
+
+                <div class="umb-control-group">
+                    <umb-button type="button"
+                                label="Mode: {{vm.selectedDeleteMode}}"
+                                action="vm.toggleDeleteModeSelect()">
+                    </umb-button>
+                    <umb-dropdown ng-if="vm.deleteModeSelectOpen" on-close="vm.deleteModeSelectOpen = false">
+                        <umb-dropdown-item ng-repeat="deleteMode in vm.deleteModes">
+                            <button type="button" class="btn-reset"
+                                    ng-click="vm.setDeleteMode(deleteMode); vm.deleteModeSelectOpen = false;">
+                                {{deleteMode}}
+                            </button>
+                        </umb-dropdown-item>
+                    </umb-dropdown>
+
+                    <umb-button type="button"
+                                action="vm.deleteFailedJobs()"
+                                button-style="danger"
+                                label="Delete"
+                                disabled="vm.deletingFailedJobs">
+                    </umb-button>
+                </div>
+            </div>
+        </div>
```

Comment:
Should we only show the delete jobs button if we have any failed jobs? And only enable the delete button if mode is 'selected' and any job is selected?

Reply:
I think it makes sense to hide it, if no failed jobs present.